### PR TITLE
Ratings

### PR DESCRIPTION
**Pull request by alirezamoosio**
Well, my fellow developers, it is time to put this branch to rest.

### CODE REVIEW

File: entities/templates/comment.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{#Comment from {{ first_name }} {{ last_name }} says {{ content }} on {{ date }}#}
```
Comment (kiarashgl) [inline entities/templates/comment.html:1]: Maybe this isn't needed anymore :-?

File: entities/templates/entities/department_detail.html [hunk 2]
```diff
@@ -31,4 +31,67 @@ <h6>
...
+{% block ratings %}
+	<br>
+	<canvas id="ratingsChart" ></canvas>
+{% endblock %}
+
+{% block custom_scripts %}
+	{{ block.super }}
+	<script>
+        Chart.defaults.global.defaultFontSize = 14;
+        $.ajax({
```
Comment (kiarashgl) [inline entities/templates/entities/department_detail.html:46]: Keep calm and Tab Bezan Soltan

File: entities/templates/entities/entity_detail.html [hunk 3]
```diff
@@ -25,15 +25,139 @@
...
 	</div>
 
+{% endblock %}
+
+{% block custom_scripts %}
+	{{ block.super }}
+	<script>
+
+        function updateLikeButtons(likeButton, dislikeButton, likes, dislikes, liked, disliked) {
+            likeButton.find(".count").text(likes);
```
Comment (kiarashgl) [inline entities/templates/entities/entity_detail.html:67]: Keep calm and Tab Bezan

File: entities/templates/entities/professor_detail.html [hunk 4]
```diff
@@ -20,4 +20,237 @@ <h6>
...
+		<div class="col-sm-4">
+			<canvas id="researchAndAdviceQChart" width="400" height="400"></canvas>
+		</div>
+	</div>
+{% endblock %}
+
+{% block custom_scripts %}
+	{{ block.super }}
+	<script>
+        Chart.defaults.global.defaultFontSize = 14;
```
Comment (kiarashgl) [inline entities/templates/entities/professor_detail.html:94]: Keep calm and Tab Bezan

File: requirements.txt [hunk 5]
```diff
@@ -9,5 +9,10 @@ django_nose==1.4.6
 coverage==5.1
 selenium==3.141.0
 webdrivermanager==0.8.0
+python-dotenv==0.13.0
```
Comment (kiarashgl) [inline requirements.txt:12]: We don't need this anymore.

File: requirements.txt [hunk 6]
```diff
@@ -9,5 +9,10 @@ django_nose==1.4.6
...
+mysqlclient==1.4.6
```
Comment (kiarashgl) [inline requirements.txt:13]: Also this.

File: entities/templates/comment.html (hunk 1 above)
Comment (alirezamoosio) [reply entities/templates/comment.html:1]: Done

File: entities/templates/entities/department_detail.html (hunk 2 above)
Comment (alirezamoosio) [reply entities/templates/entities/department_detail.html:46]: Done

File: entities/templates/entities/entity_detail.html (hunk 3 above)
Comment (alirezamoosio) [reply entities/templates/entities/entity_detail.html:67]: Done

File: entities/templates/entities/professor_detail.html (hunk 4 above)
Comment (alirezamoosio) [reply entities/templates/entities/professor_detail.html:94]: Done

File: requirements.txt (hunk 5 above)
Comment (alirezamoosio) [reply requirements.txt:12]: Done

File: requirements.txt (hunk 6 above)
Comment (alirezamoosio) [reply requirements.txt:13]: Done

**Pull request merged**